geo/Cargo.toml

```diff
@@ -34,6 +34,9 @@ use-serde = ["serde", "geo-types/serde"]
 [dev-dependencies]
 approx = "0.4.0"
 criterion = { version = "0.3" }
+# jts-test-runner is an internal crate which exists only to be part of the geo test suite.
+# As such it's kept unpublished. It's in a separate repo primarily because it's kind of large.
+jts-test-runner = { git = "https://github.com/georust/jts-test-runner", commit = "bf873f7858792498b70451c2818c40d4e0975264" }
```

Comment:
This might be controversial... but it seemed like a lot of ceremony to go through the publish dance for something which is basically an internal dependency.

I could be convinced otherwise!

Reply:
If anything, I'd reconsider just including the crate in the main geo repo - It's 15MB of xml.

Reply:
One thing to note is that the current cargo package size limit is 10mb…

Reply:
Ah, good to know! My currently solution is indeed to just agglomerate the test fixtures into the jts-test-runner crate. 

I'm including a pretty small subset at this point, but good to know that we may eventually need to do something else.

Still, I'm now more inclined to leave it as a separate crate.